src/main/java/uk/gov/hmcts/ccd/data/casedetails/query/CaseDetailsQueryBuilderFactory.java

```diff
@@ -1,6 +1,5 @@
 package uk.gov.hmcts.ccd.data.casedetails.query;
```

Comment:
the advantage of the Factory before was that it had injected the UserAuthorisationSecurity. 
if now we expose a secure method on the builder itself, which accepts the list of AuthorisationSecurity, I don't see what's the benefit anymore of the factory.
Shall we remove it? 
Up to you

Reply:
yes removed now